CartoDB/cartodb
# Using clean method instead of remove for context-menu-view

Thanks to https://github.com/CartoDB/cartodb/issues/11778.

## STTest
- Check context menus work properly in table view (in dataset and builder pages).
- Check data-layer menu works as expected (layer options in layers list).
- Check map options work as expected (where delete, export, duplicate map [etc] are placed).
- Check context menu when you want to remove a custom basemap works as expected.
